specification/resources/resource-manager/Microsoft.Solutions/stable/2019-07-01/managedapplications.json

```diff
@@ -1835,6 +2093,39 @@
       ],
       "description": "The JIT scheduling policies."
     },
+    "JitApprovalMode": {
+      "description": "The Jit approval mode.",
+      "enum": [
+        "NotSpecified",
+        "AutoApprove",
+        "ManualApprove"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "JitApprovalMode",
+        "modelAsString": true
+      }
+    },
+    "JitApproverDefinition": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The approver service principal Id."
+        },
+        "type": {
```

Comment:
enum as string with potential values

Reply:
We will return an error when in correct value is passed in. It's validated in the back end.

Reply:
The enum is for the user to discover the set of potential values, not for validation.  How do I know as a consumer of your REST API docs or the SDKs what potential values to put here?